src/Compilers/CSharp/Portable/Binder/Semantics/OverloadResolution/OverloadResolution.cs

```diff
@@ -2531,6 +2531,27 @@ private BetterResult BetterConversionFromExpression(BoundExpression node, TypeSy
             if (!conv2.IsConditionalExpression && conv1.IsConditionalExpression)
                 return BetterResult.Right;
 
+            // C1 and C2 are collection expression conversions and the following hold:
+            // - T1 is a ref struct type with iteration type E1, and T2 is a non- ref struct type with iteration type E2, and
```

Comment:
```suggestion
            // - T1 is a ref struct type with iteration type E1, and T2 is a non-ref struct type with iteration type E2, and
```

Reply:
The space was intentional so it's clear the type is not a `ref struct` rather than a `struct` that is not `ref`.

Reply:
Consider `not a "ref struct"`